airflow-core/pyproject.toml

```diff
@@ -34,7 +34,9 @@ build-backend = "hatchling.build"
 name = "apache-airflow-core"
 description = "Core packages for Apache Airflow, schedule and API server"
 readme = { file = "README.md", content-type = "text/markdown" }
-license-files.globs = ["LICENSE", "3rd-party-licenses/*.txt", "NOTICE"]
+license-expression = "Apache-2.0"
+license = "Apache-2.0"
```

Comment:
I thought we didn't need both? Or is this the difference between flit and hatchling?